fetch/handler_test.go

```diff
@@ -263,22 +261,15 @@ func TestHandleMeshHashReq(t *testing.T) {
 }
 
 func newAtx(t *testing.T, published types.EpochID) (*types.ActivationTx, types.AtxBlob) {
```

Comment:
NIT: How about returning just the ATX? The caller can create a fake blob if needed
```suggestion
func newAtx(t *testing.T, published types.EpochID) *types.ActivationTx {
```